src/libsyntax/parse/lexer/comments.rs

```diff
@@ -176,7 +176,7 @@ fn split_block_comment_into_lines(
 
 // it appears this function is called only from pprust... that's
 // probably not a good thing.
-pub fn gather_comments(sess: &ParseSess, path: FileName, src: String) -> Vec<Comment> {
+crate fn gather_comments(sess: &ParseSess, path: FileName, src: String) -> Vec<Comment> {
```

Comment:
It's not clear to me why this visibility needs to change? `with_desugared_doc` doesn't appear to need this function?

Reply:
I think this may have been a drive-by change I made at some point, but privatizing where possible is a good thing? :)